### PR TITLE
[IOTDB-1376] BatchProcessException was not correctly handled in BaseApplier

### PR DESCRIPTION
**Pull request by jt2594838**
Please refer to https://issues.apache.org/jira/browse/IOTDB-1376 for details.

### CODE REVIEW

**Comment by jt2594838** [comment]
There are duplications because Java does not support multiple extensions or non-static members in Interface, so please ignore them.

File: server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateMultiTimeSeriesPlan.java [hunk 1]
```diff
@@ -346,4 +349,38 @@ public void checkIntegrity() throws QueryProcessException {
...
+    return paths.size();
+  }
+
+  @Override
+  public void unsetIsExecuted(int i) {
+    if (isExecuted == null) {
+      isExecuted = new boolean[getBatchSize()];
+    }
+    isExecuted[i] = false;
+    if (indexes != null) {
```
Comment (mychaow) [inline server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateMultiTimeSeriesPlan.java:380]: indexs may just empty, same as above.

**Review by neuyilan — APPROVED** [review]
LGTM

File: server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateMultiTimeSeriesPlan.java (hunk 1 above)
Comment (jt2594838) [reply server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateMultiTimeSeriesPlan.java:380]: fixed

**Review by wangchao316 — APPROVED** [review]
LGTM

**Review by mychaow — APPROVED** [review]
LGTM

**Pull request merged**